test-suite/Makefile.am

```diff
@@ -122,6 +122,7 @@ QL_TEST_SRCS = \
 	quantooption.cpp \
 	quotes.cpp \
 	rangeaccrual.cpp \
+	regulatorytermstructure.cpp \
```

Comment:
`ultimateforwardtermstructure.cpp` might be a better file name&mdash;having it differ from the class name might be confusing.  Same for the hpp.

Reply:
I struggled with the naming here a bit, but I agree that the name of the file should be consistent with the name of the class.